serializers/kubernetes_cluster.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Serializers::KubernetesCluster < Serializers::Base
```

Comment:
I recommend against the creation of new serializers, especially for use in the web routes. They add unnecessary and net-negative indirection.  I recommend directly calling methods on the model objects in the web views instead of using a serializer.

Reply:
when you mention "I recommend against the creation of **new** serializers", do you mean this is an anti-pattern that we should look for abandoning in the code? If so, we can do that, but if not, similar to the previous pattern discussion, we can let it stay for the sake of obeying the pattern + preparing for the API routes of the upcoming weeks.

Reply:
Correct, I think it is an anti-pattern, at least for usage in web routes.  For web routes, we are using serializers as presenters, and a plain hash or array of hashes as a presenter object I think makes things worse.  I don't think we need such presenter objects at all, and can use the model objects directly.

For api routes, some form of serialization is necessary.  I think such serialization is probably best either inlined in the route (if only used in one route), or added as Clover helper method (if used in multiple routes).  However, I don't object to the current use of serializers for API routes.

Reply:
Alright, that's good to know. I will clean that up.